projects/plugins/jetpack/modules/gravatar-hovercards.php

```diff
@@ -135,11 +135,13 @@ function grofiles_gravatars_to_append( $author = null ) {
  * navigating to a new page
  *
  * @param string $url The comment author's URL.
+ * @param int    $id  The comment ID.
  *
  * @return string The adjusted URL
  */
-function grofiles_amp_comment_author_url( $url ) {
-	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+function grofiles_amp_comment_author_url( $url, $id ) {
+	if ( 'comment' === get_comment_type( $id ) && class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		// @todo Disabling the comment author link in this way is not ideal since clicking the link does not cause the lightbox to open in the same way as clicking the gravatar. Likely get_comment_author_url_link should be used instead so that the href attribute can be replaced with an `on` attribute that activates the gallery.
```

Comment:
This can be addressed in a subsequent PR.

Reply:
The Gravatar Hovercards implementation in AMP doesn't seem ideal:

1. It doesn't seem like the author link should be overridden at all, since the link in the non-AMP version is not overridden. 
2. The AMP lightbox to show the avatar is huge. It should be much smaller to be appropriate for the size of the avatar.
3. The image in the lightbox shouldn't be linked with other images on the page. There's no need to cycle through them all.